src/main/java/customer/CustomerList.java

```diff
@@ -0,0 +1,36 @@
+package customer;
+
+import java.util.ArrayList;
+
+public class CustomerList {
+
+    public static ArrayList<Customer> customers = new ArrayList<>();
```

Comment:
Inconsistent naming through the codebase: customers v.s. carsList.
It would be good if this is consistent throughout the codebase.